sql/field.h

```diff
@@ -1227,6 +1237,7 @@ class Field
   void set_column_format(column_format_type column_format_arg)
   {
     DBUG_ASSERT(column_format() == COLUMN_FORMAT_TYPE_DEFAULT);
+    flags &= ~(FIELD_FLAGS_COLUMN_FORMAT_MASK);
```

Comment:
Remove space before the assignment operator (or keep it to keep consistency with the next line; up to you)